packages/xrpl/src/models/methods/ammInfo.ts

```diff
@@ -0,0 +1,81 @@
+import { Amount, IssuedCurrencyAmount } from '../common'
+
+import { BaseRequest, BaseResponse } from './baseMethod'
+
+/**
+ * The `amm_info` command retrieves information about an AMM instance.
+ * Returns an {@link AMMInfoResponse}.
+ *
+ * @category Requests
+ */
+export interface AMMInfoRequest extends BaseRequest {
+  command: 'amm_info'
+
+  /**
+   * A hash that uniquely identifies the AMM instance.
+   */
+  amm_id?: string
+
+  /**
+   * Specifies one of the pool assets (XRP or token) of the AMM instance.
+   */
+  asset1?: Amount
+
+  /**
+   * Specifies the other pool asset of the AMM instance.
+   */
+  asset2?: Amount
+}
+
+/**
+ * Response expected from an {@link AMMInfoRequest}.
+ *
+ * @category Responses
+ */
+export interface AMMInfoResponse extends BaseResponse {
+  result: {
+    /**
+     * The account that tracks the balance of LPTokens between the AMM instance via Trustline.
+     */
+    AMMAccount: string
+
+    /**
+     * Specifies one of the pool assets (XRP or token) of the AMM instance.
+     */
+    Asset1: Amount
+
+    /**
+     * Specifies the other pool asset of the AMM instance.
+     */
+    Asset2: Amount
+
+    /**
+     * Represents the liquidity providers' shares of the AMM instance's pools.
+     * LPTokens are tokens on XRPL. Each LPToken represents a proportional share of each pool of the AMM instance.
+     * The AMM instance account issues the LPTokens to LPs upon liquidity provision.
+     * LPTokens are balanced in the LPs trustline upon liquidity removal.
```

Comment:
What does it mean for LPTokens to be balanced in the LPs trustline? (Does it remove tokens if they stop providing liquidity?)